### PR TITLE
README: modify Quick Start for docker build

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. Mark menuconfig and clean/distclean as optional steps
   with sub-title because they are not essential steps.
  essential steps : install docker -> clone source code -> configure -> build -> program
  optionals : menuconfig and clean

2. Add the program step.
  This step introduces a programming.

3. Modify content of clone and clean tabs.

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [dd4184029359d7f664f6ea38ac200b12970e1e8c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b58a033ed0b777e65d159df379e24dcaa1a7ecac] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b58a033ed0b777e65d159df379e24dcaa1a7ecac] - Code Rule Check (C++) OK.

**Comment by rzr** [comment]
I guess it should be able to flash from docker too , I also made my dockerfiles I never upstreamed 
my blocking point was that I needed the flasher too from artik sdk
since it may be public elsewhere
I will check this out later

File: README.md [hunk 1]
```diff
@@ -32,64 +32,74 @@ git clone https://github.com/Samsung/TizenRT.git
 cd TizenRT
 TIZENRT_BASEDIR="$PWD"
 ```
-**Note**: To contribute in this community, fork this project first into own github repository and do git cloning from your repository. 
-> git clone https://github.com/your_github_id/TizenRT.git
+**Note**: To contribute in this community, you need to clone your forked private repository instead of above.  
+          Github guides it by [working-with-forks](https://help.github.com/articles/working-with-forks).
```
Comment (davidfather) [inline README.md]: Github guides this.

**Comment by seinfra** [comment]
Target : [7c953522e510e638165c6046bb00d2d7ce690ad8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7c953522e510e638165c6046bb00d2d7ce690ad8] - Code Rule Check (C++) OK.

**Pull request merged**